neuro-cli/src/neuro_cli/admin.py

```diff
@@ -53,6 +53,18 @@ async def get_clusters(root: Root) -> None:
         root.print(fmt(clusters.values()))
 
 
+@command(hidden=True)
```

Comment:
Is there reason to make the command hidden?

Reply:
This command is mostly the same as `get-clusters` and can return different data only for clusters that failed to set up properly on the platform-config side. It is very low-level and can be useful to debug very strange situations. So I prefer to hide it from normal users as it is hard to explain why we need this command.

Reply:
Understood